acceptance/steps/k8s.go

```diff
@@ -16,6 +16,11 @@ import (
 	redpandav1alpha2 "github.com/redpanda-data/redpanda-operator/operator/api/redpanda/v1alpha2"
 )
 
+// this is a nasty hack due to the fact that we can't disable the linter for typecheck
```

Comment:
😬 hadn't seen that issue before

Reply:
upgrading golangci-lint is generally pretty painful punt that to another day / week / month 😛